sources/packages/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -135,7 +138,8 @@ export class ApplicationStudentsController extends BaseController {
   @ApiOkResponse({ description: "Application submitted." })
   @ApiUnprocessableEntityResponse({
     description:
-      "Program Year is not active or invalid study dates or selected study start date is not within the program year or APPLICATION_NOT_VALID or INVALID_OPERATION_IN_THE_CURRENT_STATUS or ASSESSMENT_INVALID_OPERATION_IN_THE_CURRENT_STATE.",
+      "Program Year is not active or invalid study dates or selected study start date is not within the program year or APPLICATION_NOT_VALID or INVALID_OPERATION_IN_THE_CURRENT_STATUS or ASSESSMENT_INVALID_OPERATION_IN_THE_CURRENT_STATE " +
```

Comment:
I would prefer to have this long string a bit more broken in the same way that we have in other places.
I saw that it was not introduced in this PR, but if possible just copy and paste from the below sample it would look better, IMO.
```ts
@ApiUnprocessableEntityResponse({
    description:
      "Program Year is not active or " +
      "invalid study dates or selected study start date is not within the program year" +
      "or APPLICATION_NOT_VALID or INVALID_OPERATION_IN_THE_CURRENT_STATUS or ASSESSMENT_INVALID_OPERATION_IN_THE_CURRENT_STATE " +
      "or OFFERING_NOT_VALID.",
  })
```